test/tools/systemMemory/README.md

```diff
@@ -0,0 +1,21 @@
+# System Memory
+Get physical memory information via `chrome.system.memory.getInfo()` method, and it gathers memory information per second (in MB). 
+
```

Comment:
Thanks @Christywl ! Why do we use "systemMemory" rather than "SystemMemory"?

Reply:
sorry, I forgot the first letter, I'll update the path name